src/MiniCover/Commands/Options/MiniCoverOption.cs

```diff
@@ -5,8 +5,10 @@ namespace MiniCover.Commands.Options
 {
     public abstract class MiniCoverOption<TValue> : IMiniCoverOption<TValue>
     {
-        protected bool Validated;
-        protected TValue ValueField;
+        private CommandOption option;
```

Comment:
Moved option storage and knowledge to the base class, to reduce code inside all options.